src/applications/simple-forms/mock-simple-forms-patterns/config/form.js

```diff
@@ -253,6 +260,13 @@ const formConfig = {
           arrayPath: 'employers',
```

Comment:
if you look at the overall way the array pages are defined here, it's still pretty verbose, but at least now we have options for everything. 
- onNavBack/onNavForward: we need to control what happens on back and on continue (for example we might need to remove an item on pressing back, or keeping the add=true urlParams when going to the next page)
- ContentBeforeButtons for adding a cancel button that changes routes after a modal

goal will be to simplify this further once everything's established